app/src/js/components/Map.js

```diff
@@ -45,12 +46,20 @@ export function Map(props) {
     setMap(map);
     props.onMapInit(map);
 
-    const interval = setInterval(() => {
+    const styleLoadedInterval = setInterval(() => {
       if (map.isStyleLoaded() && !styleLoaded) {
         setStyleLoaded(true);
       }
     }, 100);
-    return () => clearInterval(interval);
+
+    const focusInterval = setInterval(() => {
+      setFocusBlink(focusBlink => !focusBlink);
+    }, 500);
+
+    return () => {
```

Comment:
could us not doing this be the cause of the react error we see in Notifications?

Reply:
it was!